### PR TITLE
Implement stars in community list

### LINKED ISSUE #348: Communities List Should Have the Ability to Favorite by Tapping a Star
Loving all the work that has gone into Voyager! Is there any way you could add the option to favorite communities from the Posts > Communities area by tapping a Star, similar to Apollo?

I’ve mocked up what it would look like. image

### PR DESCRIPTION
**Pull request by rsammelson**
Fixes #348

### CODE REVIEW

File: src/pages/posts/CommunitiesPage.tsx [hunk 1]
```diff
@@ -94,7 +100,12 @@ export default function CommunitiesPage() {
     [communities, favorites]
   );
 
+  const dispatch = useAppDispatch();
+  const [present] = useIonToast();
+
   function renderCommunity(community: Community) {
+    const isFavorite = favorites.includes(getHandle(community));
```
[the remark this answers is not in the document]
Comment (rsammelson) [reply src/pages/posts/CommunitiesPage.tsx:107]: Yeah I had kind of wanted to memoize that, I will refactor it.

**Comment by rsammelson** [comment]
@aeharding I fixed the memoization.

**Comment by aeharding** [comment]
Hi @rsammelson I have some adjustments but I'm having trouble pushing to this branch

**Comment by aeharding** [comment]
The changes I pushed to a local branch here: https://github.com/aeharding/voyager/tree/community-list-favorites

**Comment by rsammelson** [comment]
Fixed it, you should be able to push now

**Comment by aeharding** [comment]
Great! Nice work here by the way. This is a much better way to add favorites imo

**Pull request merged**